File: settlement/room/law/PUNISHMENT_SERVICE.java
package settlement.room.law;

import init.race.Race;
import util.data.BOOLEANO.BOOLEAN_OE;

public interface PUNISHMENT_SERVICE {

	public int punishTotal();
	public int punishUsed();

	public default BOOLEAN_OE<Race> punishEnabled(){
		return null;
	}
	
	
	
}

File: LegacySource/settlement/room/law/PUNISHMENT_SERVICE.cs
using System;
using init.race;
using util.data.BOOLEANO;

namespace settlement.room.law
{
    public interface PUNISHMENT_SERVICE
    {
        public int punishTotal();
        public int punishUsed();

        public default BOOLEAN_OE<Race> punishEnabled()
        {
            return null;
        }
    }
}